sources/packages/api/src-sql/Reports/Create-disbursement-report.sql

```diff
@@ -0,0 +1,44 @@
+--Insert config for disbursement report.
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Disbursement_Report',
+        'SELECT "Date of Disbursement",
+            "SIN",
+            "Application Number",
+            "Certificate Number",
+            "Funding Code",
+            "Disbursement amount"
+          FROM
+            (SELECT TO_CHAR(DR.DISBURSE_DATE, ''YYYY-MM-DD'') AS "Date of Disbursement",
+                DR.STUDENT_SIN AS "SIN",
+                SA.APPLICATION_ID AS "Application Number",
```

Comment:
The query is not following the same format as the previous one already in place. For instance, this one is using the DB columns/aliases in all upper cases while the previous ones are not.